crates/common/consensus/lean/src/state.rs

```diff
@@ -284,11 +304,15 @@ impl LeanState {
     }
 
     pub fn process_attestations(&mut self, attestations: &[Attestation]) -> anyhow::Result<()> {
```

Comment:
We're adding metrics to `state.process_attestations()` but I think leanMetrics want attestation valid/invalid on fork choice's on validate attestation. I believe state operations happen a lot more often e.g. when building a block, while fork choice on_attestation happens only when receiving a new block, so the number will be skewed.

Can you double check with leanMetrics again?